src/parser.js

```diff
@@ -1857,7 +1894,7 @@ export class GenericParser extends Tokenizer {
     if (this.lookahead.value === 'yield' && this.allowYieldExpression) {
       throw this.createError(ErrorMessages.ILLEGAL_YIELD_IDENTIFIER);
     }
-    if (this.lookahead.value === 'await' && this.allowAwaitExpression) {
+    if (this.lookahead.value === 'await' && (this.allowAwaitExpression)) {
```

Comment:
why this?

Reply:
I don't know?

Reply:
@Shilpi3 just remove the extra parentheses so it's how it was.